recipes/mapgl/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mapGL" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f053573cba3f5482620d68464f6fd6d5505e0e82bbcffd8e65d43815829c86d6"
+
+build:
+  number: 0
+  entry_points:
+    - mapGL.py = map_GL.mapGL:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: generic
+
+requirements:
+  host:
+    - pathlib
+    - bx-python
+    - numpy
```

Comment:
Does it really need, pathlib only during installation? Does it link against numpy or bx-python? If not, please remove them from `host:`.

Reply:
Retrying without pathlib.